src/riak_kv_crdt.erl

```diff
@@ -132,6 +134,19 @@ map_value(RObj) ->
     {{_Ctx, Map}, _Stats}  = value(RObj, ?MAP_TYPE),
     Map.
 
+%% @doc convenience function for (e.g.) Yokozuna. Checks the bucket props for
+%% the object, if it has a supported datatype entry, returns true; otherwise
+%% false if not a 2.0 CRDT.
+-spec is_crdt(riak_object:riak_object()) -> boolean().
+is_crdt(RObj) ->
+    Bucket = riak_object:bucket(RObj),
+    case riak_core_bucket:get_bucket(Bucket) of
```

Comment:
Looking at other spots, e.g. https://github.com/basho/riak_kv/blob/ec7e525fd3af7e6324474e8ea8949626c5c8b061/src/riak_kv_util.erl#L163, I decided to keep the case, but actually have an error case if BProps is not a list for some reason.